engine/clients/pgvector/search.py

```diff
@@ -0,0 +1,55 @@
+import multiprocessing as mp
+from typing import List, Tuple
+
+import numpy as np
+import psycopg2
+from pgvector.psycopg2 import register_vector
+from psycopg2.extras import RealDictCursor
+
+from engine.base_client.distances import Distance
+from engine.base_client.search import BaseSearcher
+from engine.clients.pgvector.config import get_db_config
+from engine.clients.pgvector.parser import PgVectorConditionParser
+
+
+class PgVectorSearcher(BaseSearcher):
+    search_params = {}
+    cursor = None
+    parser = PgVectorConditionParser()
+
+    @classmethod
+    def get_mp_start_method(cls):
+        return "forkserver" if "forkserver" in mp.get_all_start_methods() else "spawn"
+
+    @classmethod
+    def init_client(cls, host, distance, connection_params: dict, search_params: dict):
+        cls.conn = psycopg2.connect(**get_db_config(host))
```

Comment:
`cls.conn` should be defined as a class member (check how it is implemented in QdrantSearcher)

Reply:
Done